### PR TITLE
#74/#75 - hashing/style - michellejanay

### LINKED ISSUE #74: restyle login and signup pages (logo and font-size)
As a developer,
I want to see the login and signup pages on one page
So that it is more user friendly 

AC1: Make Logo smaller to fit on page (add a new class for those pages)
AC2: Change font-size, ect, to make the forms smaller

### LINKED ISSUE #75: Password Hashing
As a user,
I want to be sure that my passwords are secure
So that I won't have to worry about data leaks in my account

AC1: find a secure hashing library and implement for user passwords

### PR DESCRIPTION
**Pull request by michellejanay**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #74 
Fixes #75 

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/Components/LoginPage/LoginForm.js [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
-      <label htmlFor='email'>Email</label>
-      <input type="email" value={email} onChange={handleEmailChange} className="userInput" id='email' />
-      <label htmlFor='password' >Password</label>
-      <input type="password" value={password} onChange={handlePasswordChange} className="userInput" id='password'/>
-      <button type="submit" className='submitBtn' >Continue</button>
-     </form>
-   </div>
-  )
-}
+		console.log({ email: email, password: password });
```
Comment (farzad-nazif) [inline client/src/Components/LoginPage/LoginForm.js]: For security purposes, it's better to not have console.log in the code. It's good for debugging.

File: client/src/Components/SignUpPage/SignUpForm.js [hunk 2]
```diff
@@ -1,77 +1,116 @@
...
+	const onRadioSelect = (e) => {
+		setSelected(e.target.value);
+	};
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		const email = emailInputRef.current.value;
+		const password = bcrypt.hashSync(passwordInputRef.current.value, 10);
+
+		fetch("/signUp", {
```
Comment (farzad-nazif) [inline client/src/Components/SignUpPage/SignUpForm.js:22]: I like how you make it ready for backend. Well done!

File: client/src/Components/LoginPage/LoginForm.js (hunk 1 above)
Comment (farzad-nazif) [reply client/src/Components/LoginPage/LoginForm.js]: Please if possible, remove this.

Comment (michellejanay) [reply client/src/Components/LoginPage/LoginForm.js]: Ah yes, I forgot to remove them. Thank you!

**Review by ChidimmaOfodum — APPROVED** [review]

**Pull request merged**